nmxact/udp/udp_sesn.go

```diff
@@ -118,6 +118,11 @@ func (s *UdpSesn) TxRxMgmt(m *nmp.NmpMsg,
 	return s.txvr.TxRxMgmt(txRaw, m, s.MtuOut(), timeout)
 }
 
+func (s *UdpSesn) TxRxMgmt_async(m *nmp.NmpMsg,
+	timeout time.Duration, ch chan nmp.NmpRsp, err_c chan error) error {
+	return nil
```

Comment:
It might be better to return `fmt.Errorf("unsupported")` or similar here.

Reply:
done.